hooks/useVotingPlugins.ts

```diff
@@ -60,70 +70,108 @@ export function useVotingPlugins() {
     handleSetPythClient,
     handleSetCurrentRealmVotingClient,
   } = useVotePluginsClientStore()
-  const {
-    setVotingNfts,
-    setMaxVoterWeight,
-    setIsLoadingNfts,
-  } = useNftPluginStore()
-  const {
-    setIsLoadingGatewayToken,
-    setGatekeeperNetwork,
-  } = useGatewayPluginStore()
-  const {
-    setIsLoading,
-    setVotingPower,
-    setOracleKeys,
-    setInstructions,
-  } = useSwitchboardPluginStore()
 
+  const nftStore = useNftPluginStore()
+  const heliumStore = useHeliumVsrStore()
+  const gatewayStore = useGatewayPluginStore()
+  const switchboardStore = useSwitchboardPluginStore()
```

Comment:
good

Reply:
i hate this file

Reply:
would it be possible to make all Helium stuff live in its own hook, which just returns `handleHeliumBlahblah` and then have this abominable hook simply use that?

Reply:
Yeah this file is rough, that could be possible yeah. Is that something that would prevent this pr?

Reply:
No its not a blocker, nothing regarding code quality really is much of a blocker.